xarray/core/dataset.py

```diff
@@ -283,6 +287,18 @@ def attrs(self, value):
         self._attrs = OrderedDict(value)
 
     @property
+    def encoding(self):
+        """Dictionary of global encoding attributes on this dataset
+        """
+        if self._encoding is None:
+            self._encoding = OrderedDict()
```

Comment:
I would just make it a normal dict, like `Variable.encoding`, unless you have a reason why you feel preserving order would be helpful.